cartography/data/indexes.cypher

```diff
@@ -114,3 +114,6 @@ CREATE INDEX ON :S3Bucket(id);
 CREATE INDEX ON :S3Bucket(name);
 CREATE INDEX ON :S3Bucket(arn);
 CREATE INDEX ON :User(arn);
+CREATE INDEX ON :AzureTenant(id);
+CREATE INDEX ON :AzurePrincipal(email);
+CREATE INDEX ON :AzureSubscription(id);
```

Comment:
We need to add indexes for all sql related entities here

Reply:
Done